dataframe_api_compat/pandas_standard/column_object.py

```diff
@@ -35,6 +35,7 @@
     "UInt16": "uint16",
     "UInt8": "uint8",
     "boolean": "bool",
+    "Float64": "float64",
```

Comment:
I also discovered by accident, it seems that the float type was missing, but if it was done on purpose, I can try to redo it.

Reply:
i probably just forgot it - let's add float32 too?

Reply:
added